RiotSwiftUI/Modules/Room/LocationSharing/Coordinator/LocationSharingCoordinator.swift

```diff
@@ -101,19 +101,15 @@ final class LocationSharingCoordinator: Coordinator, Presentable {
         }
     }
     
+    static func shareActivityControllerForLocation(_ location: CLLocationCoordinate2D) -> UIActivityViewController {
```

Comment:
Only a thought, not important.
```suggestion
    static func shareLocationActivityController(_ location: CLLocationCoordinate2D) -> UIActivityViewController {
```

Reply:
No no, naming **is** important. I think your is nicer.